src/aws.js

```diff
@@ -11,7 +11,7 @@ AWS.config.update({
 });
 const S3 = new AWS.S3();
 
-const upload = (client, params) => {
+const upload = async(client, params) => {
```

Comment:
Doesn't need to be `async`:  a function needs to be `async` if you use `await` in the body, not if you `await` on the function itself. You can `await` any promise, it doesn't matter if they're explicitly created or implicitly by using an `async` function